website/docs/notes/concepts.md

```diff
@@ -21,52 +19,71 @@ To achieve this, MMF has few opinions about architecture of your research projec
 
```

Comment:
This needs to be updated. There is no concept of tasks anymore, there are just dataset. No need to also show the table below. Point to dataset list instead.

Reply:
Update the TOC with proper links as well.